roles/DCOS.agent_windows/files/start.ps1

```diff
@@ -0,0 +1,17 @@
+$env:HostIP = (
+    Get-NetIPConfiguration |
+    Where-Object {
+        $_.IPv4DefaultGateway -ne $null -and
+        $_.NetAdapter.Status -ne "Disconnected"
+    }
+).IPv4Address.IPAddress
+
+$logpath = "c:\mesos-logs\"
```

Comment:
this this should be read from ENV var with a default. As mentioned in other commands I think we should keep everything in a dcos folder and not on root e.g. `/dcos/logs`

Reply:
👍

Reply:
removed the file